src/components/collection/CollectionPlayerContainer.jsx

```diff
@@ -26,20 +27,21 @@ const CollectionPlayerContainer = ({
   // Helper fn to get segements
   const getSegments = useCallback((i) => collection.tracks[i].segments, [collection])
   const getId = useCallback((i) => collection.tracks[i].id, [collection])
+  const getGateways = useCallback((i) => formatGateways(collection.gateways))
```

Comment:
it looks like we don't actually use `i` anywhere?

Reply:
oh this is wrong. it shhould be collection.tracks[i]. good catch